Apps/ValidationTests/Win32/App.cpp

```diff
@@ -9,7 +9,10 @@
 #include <Windowsx.h>
 #include <Shlwapi.h>
 #include <filesystem>
+#include <algorithm>
 
+bool doExit = false;
```

Comment:
No window on CI and I need to exit properly with an error. It's the simplest way to do it.